<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartScene</class>
 <widget class="QMainWindow" name="StartScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>开始复习您的日语假名</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/res/kanas_review.ico</normaloff>:/res/kanas_review.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionBgm_Bg_Setting"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/Exit.png</normaloff>:/res/Exit.png</iconset>
   </property>
   <property name="text">
    <string>退出系统</string>
   </property>
  </action>
  <action name="actionBgm_Bg_Setting">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/settingIcon.png</normaloff>:/res/settingIcon.png</iconset>
   </property>
   <property name="text">
    <string>设置背景音乐和背景</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>BgSetting</string>
   </property>
  </action>
  <action name="actionchooseScene">
   <property name="text">
    <string>chooseScene</string>
   </property>
  </action>
  <action name="actionexeScene">
   <property name="text">
    <string>exeScene</string>
   </property>
  </action>
  <action name="actionexeReport">
   <property name="text">
    <string>exeReport</string>
   </property>
  </action>
  <action name="actiontestCreate">
   <property name="text">
    <string>testCreate</string>
   </property>
  </action>
  <action name="actionsetting">
   <property name="text">
    <string>setting</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/helpIcon.png</normaloff>:/res/helpIcon.png</iconset>
   </property>
   <property name="text">
    <string>查看帮助</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
